app/src/lumen/res/layout/history_viewholder.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center">
+
+    <RelativeLayout
```

Comment:
Avoid `RelativeLayout`

Reply:
I copied this file from cliqz. And here it looks like we need the children to be arranged in relation to one another.

Reply:
I did it myself